API/tasks/getBalance.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function(web3) {
+
+	return ({
+		/**
+		 * Get Balance
+		 * @param {String} address - The user's pub key
+		 * @param {Array} tokens - The tokens to be fetched
+		 * @return {Array} - Balances of that user
+		 */
+
+		/**
+		 * Represents a users balance
```

Comment:
this js doc only needs to detail the getBalance function

Reply:
done